features/print.yml

```
@@ -0,0 +1,8 @@
+name: Printing
+description: The `window.print()` method, along with the `beforeprint` and `afterprint` events, allow you to change the page for printing, open the browser's print dialog, and restore the page after printing.
```

Should we say that `@media print` can also be used to adapt the page for printing?

---

Honestly, I think we should _recommend_ `@media print`. 😄 We could be a little opinionated here and say something like:

```suggestion
description: An alternative to `@media print` queries, the `window.print()` method, along with the `beforeprint` and `afterprint` events, allow you to change the page for printing, open the browser's print dialog, and restore the page after printing.
```

Or stronger still:

```suggestion
description: The `window.print()` method, along with the `beforeprint` and `afterprint` events, allow you to change the page for printing, open the browser's print dialog, and restore the page after printing. Consider using `@media print` queries instead.
```

---

I think the first is strong enough. I'm sure there are cases where CSS alone just can't do the job, or where it's clearly the wrong tradeoff.